spark/src/main/kotlin/com/adevinta/spark/components/popover/Popover.kt

```diff
@@ -0,0 +1,179 @@
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import com.adevinta.spark.ExperimentalSparkApi
+import com.adevinta.spark.PreviewTheme
+import com.adevinta.spark.SparkTheme
+import com.adevinta.spark.SparkTheme.colors
+import com.adevinta.spark.components.buttons.ButtonOutlined
+import com.adevinta.spark.components.iconbuttons.IconButtonGhost
+import com.adevinta.spark.components.popover.PopoverDefaults.PopoverAnchorPadding
+import com.adevinta.spark.components.popover.PopoverDefaults.PopoverContentPadding
+import com.adevinta.spark.components.popover.PopoverDefaults.PopoverDismissButtonPadding
+import com.adevinta.spark.components.popover.PopoverDefaults.PopoverMinHeight
+import com.adevinta.spark.components.popover.PopoverDefaults.PopoverMinWidth
+import com.adevinta.spark.components.popover.newapi.RichTooltipColors
+import com.adevinta.spark.components.popover.newapi.TooltipBox
+import com.adevinta.spark.components.popover.newapi.TooltipDefaults
+import com.adevinta.spark.components.popover.newapi.TooltipState
+import com.adevinta.spark.components.popover.newapi.rememberTooltipState
+import com.adevinta.spark.components.surface.Surface
+import com.adevinta.spark.components.text.Text
+import com.adevinta.spark.icons.Close
+import com.adevinta.spark.icons.SparkIcons
+import com.adevinta.spark.tokens.ElevationTokens
+import com.adevinta.spark.tools.preview.ThemeProvider
+import com.adevinta.spark.tools.preview.ThemeVariant
+import kotlinx.coroutines.launch
+
+/**
+ * Popover is kinda similar to Plain tooltip from Compose UI
+ * Provides a descriptive message or Info for an Anchor.
+ * Popover that is invoked when the anchor is pressed/long pressed:
+ *
+ * Params:
+ * @param popoverContent the composable that will be used to populate the Popover's content.
+ * @param isDismissButtonEnabled [Boolean] that determines if we show a dismiss iconbutton on the Popover,
+ * @param popoverState handles the state of the Popover's visibility.
+ * @param actionContent the composable that the Popover will anchor to.
+ */
+
+@ExperimentalSparkApi
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+@SuppressLint("ComposeModifierMissing")
+public fun Popover(
+    popoverContent: @Composable () -> Unit,
+    isDismissButtonEnabled: Boolean = false,
+    popoverState: TooltipState = rememberTooltipState(isPersistent = true),
+    actionContent: @Composable () -> Unit,
+) {
+    val shape: Shape = TooltipDefaults.richTooltipContainerShape
+
+    TooltipBox(
+        positionProvider = TooltipDefaults.rememberPlainTooltipPositionProvider(),
+        tooltip = {
+            val colors: RichTooltipColors = TooltipDefaults.richTooltipColors()
+            Surface(
+                modifier = Modifier
+                    .sizeIn(minWidth = PopoverMinWidth, minHeight = PopoverMinHeight)
+                    .padding(bottom = PopoverAnchorPadding),
+                shape = shape,
```

Comment:
```suggestion
                shape = shape,
                elevation = ElevationTokens.Level2,
```
We still need the elvation to be applied 😉